src/templates/en/2019/base_chapter.html

```diff
@@ -119,6 +127,7 @@
 
 {% macro render_prevnext() %}
   {% if prev_chapter %}
+test
       <a id="previous-chapter" href="/{{lang}}/{{year}}/{{prev_chapter['title'].lower().replace(' ', '-').replace('/', '')}}">
```

Comment:
Could we add a title to show the shortcuts on hover? And similarly on next button?

```suggestion
      <a title="Previous Chapter (press 'p' or ',')" id="previous-chapter" href="/{{lang}}/{{year}}/{{prev_chapter['title'].lower().replace(' ', '-').replace('/', '')}}">
```